packages/story-editor/src/app/currentUser/currentUserProvider.tsx

```diff
@@ -68,13 +79,9 @@ function CurrentUserProvider({ children }) {
       toggleWebStoriesMediaOptimization,
       updateCurrentUser,
     },
-  };
+  } as CurrentUserState;
```

Comment:
Using `as` is dangerous because it forces TS to just treat it as this type even if it doesn't actually match, hiding potential errors.

Should use `const state: CurrentUserState = {...}` here.

Then you'll see that the types don't match quite yet (see my other comment for resolving that)